.github/workflows/build.yml

```diff
@@ -6,6 +6,7 @@ on:
       - main
   schedule:
     - cron: "30 9 * * *"
+    - cron: "5 16 * * 3,4"
```

Comment:
Why the last 3,4?

Reply:
Wednesday and Thursdays, the possible days for webinars